theano/scan_module/tests/test_scan.py

```diff
@@ -1772,8 +1777,16 @@ def reset_rng_fn(fn, *args):
         analytic_grad = reset_rng_grad_fn(v_u, v_x0, vW_in)
         utt.assert_allclose(analytic_grad[0][:2], numpy.zeros((2, 2)))
 
-    @attr('slow')
```

Comment:
I'm not sure what the threshold is to tag a test as "slow", so maybe we should keep it?